ComponentKit/Core/CKComponent.mm

```diff
@@ -143,7 +143,7 @@ - (CKComponentViewContext)viewContext
     [v setBounds:{v.bounds.origin, size}];
 
     _mountInfo->viewContext = {v, {{0,0}, v.bounds.size}};
-    return {.mountChildren = YES, .contextForChildren = effectiveContext.childContextForSubview(v)};
+    return {.mountChildren = YES, .contextForChildren = effectiveContext.childContextForSubview(v, g.blockedAnimations)};
```

Comment:
Über-nit, but when I read this line I thought for a second that `blockedAnimations` was a list of all the animations that were blocked--which I thought would be super cool. I had to look at the header file to remember that it was a boolean indicating whether animations were blocked. Maybe this could use a better name?

(I realize that I reviewed #254, which added it in the first place. :stuck_out_tongue:)